cern_access/indico_cern_access/migrations/20241015_1702_92377810f14e_modify_reservation_code_field.py

```diff
@@ -0,0 +1,29 @@
+"""Modify reservation code field
+
+Revision ID: 92377810f14e
+Revises: ee88b64f9494
+Create Date: 2024-10-15 17:02:59.886902
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+
+# revision identifiers, used by Alembic.
+revision = '92377810f14e'
+down_revision = 'ee88b64f9494'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_unique_constraint(
+        'uq_reservation_code', 'access_requests', ['reservation_code'],
+        schema='plugin_cern_access')
```

Comment:
```suggestion
    op.create_unique_constraint(None, 'access_requests', ['reservation_code'], schema='plugin_cern_access')
```

the name is autogenerated by the sqlalchemy naming policy in the core